magnum_cluster_api/resources.py

```diff
@@ -1192,7 +1196,7 @@ def get_object(self) -> objects.Cluster:
                                 utils.get_cluster_label(
                                     self.cluster,
                                     "calico_ipv4pool",
-                                    "10.100.0.0/16",
+                                    "10.200.0.0/16",
                                 )
                             ],
                         },
```

Comment:
according to this:

https://cluster-api.sigs.k8s.io/tasks/bootstrap/kubeadm-bootstrap.html#kubeadmconfig-objects

we should add the following:

```python
"service": {
  "cidrBlocks": [
    utils.get_cluster_label(
      self.cluster,
      "service_cluster_ip_range",
      "10.254.0.0/16"
    )
  ]
}
```